src/Microsoft.ML.Tokenizers/Model/TiktokenTokenizer.cs

```diff
@@ -1209,7 +1221,15 @@ private static TiktokenTokenizer CreateForModel(
                     tiktokenConfiguration.VocabFile,
                     out (Dictionary<ReadOnlyMemory<byte>, int> encoder, Dictionary<StringSpanOrdinalKey, (int Id, string Token)> vocab, Dictionary<int, ReadOnlyMemory<byte>> decoder) cache))
             {
-                using Stream compressedStream = typeof(Tokenizer).Assembly.GetManifestResourceStream(tiktokenConfiguration.VocabFile)!;
+                Debug.Assert(tiktokenConfiguration.TypeName is not null && tiktokenConfiguration.PackageName is not null);
+
+                Type? type = Type.GetType($"{tiktokenConfiguration.TypeName}, {tiktokenConfiguration.PackageName}");
```

Comment:
Should we include the full assembly name here?  Not sure if there's potentially a versioning issue and in-proc side by side EG: someone has completely separate 4.0 and 5.0 usage in the same process.  I think we can just use the same version & PKT info from the current assembly.